go/vumitools/billing_worker.py

```diff
@@ -251,8 +261,10 @@ def process_outbound(self, config, msg, connector_name):
                 log.info(
                     "Not billing for outbound message: %r" % msg.to_json())
             else:
-                yield self.create_transaction_for_outbound(msg)
+                transaction = yield self.create_transaction_for_outbound(msg)
                 msg_mdh.set_paid()
```

Comment:
``transaction`` is probably a bad name for the result because the transaction data is not only one part of the result?